iosApp/ci_scripts/ci_post_clone.sh

```diff
@@ -89,4 +89,4 @@ touch .envrc
 echo "export SENTRY_DSN_IOS=${SENTRY_DSN}" >> .envrc
 echo "export SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}" >> .envrc
 echo "export FIREBASE_KEY=${FIREBASE_KEY}" >> .envrc
-echo "export GOOGLE_APP_ID=${GOOGLE_APP_ID}" >> .envrc
+echo "export GOOGLE_APP_ID_IOS=${GOOGLE_APP_ID_IOS}" >> .envrc
```

Comment:
We could preserve backwards compatibility with the Xcode Cloud workflow with `export GOOGLE_APP_ID_IOS=${GOOGLE_APP_ID}` but I think that that could get confusing.

Reply:
Actually, though, that might be a good idea so that we can merge this PR before changing the Xcode Cloud workflow instead of breaking the Xcode Cloud workflow for other PRs while this one is still in flight.